### ISSUE TITLE
Support for Bundling Entire Directories

### ISSUE THREAD

**Issue by EhanAhamed** [EhanAhamed/webCompounder#1]
Currently directories are not supported as inputs because of the `readInput` function in `./bin/index.js` (Line 78-84 as of right now)

```javascript
function readInput(input, callback) {
  async.mapSeries(input, readFile, callback);

  function readFile(input, callback) {
    fs.readFile(input, callback);
  }
}
```

The problem is that this method assumes every input is a file, and runs `fs.readFile()` on each string in the input array.

To fix this I can create a seperate function that runs before `readInput` and `writeOutput` which runs through every input and checks if its a file or directory. If its a file, it adds it to a new array. If its a directory it runs a function that returns an array of all files and directories in the directory and merges it with the prevously mentioned array.

Once the new array is constructed, it will check if there are any directories remaining, if there are, it repeats the process until there isnt.

Once there isnt, it passes the new array into the `async.waterfall(`. (Line 62 as of right now)

This is theoretical and just basically a way for me to organize my thoughts (and a socially acceptable way to talk to myself)

**Comment by EhanAhamed** [comment]
Another option is to instead pass the input array through another dependency such as globby which does this file finding itself instead of having to reinvent the wheel. Although both options seem to be fairly simple.

**Comment by EhanAhamed** [comment]
Fixed in v1.1.5
